browser/extensions/api/brave_shields_api.cc

```diff
@@ -295,5 +322,12 @@ BraveShieldsGetNoScriptControlTypeFunction::Run() {
   return RespondNow(OneArgument(std::move(result)));
 }
 
+ExtensionFunction::ResponseAction
+BraveShieldsOnShieldsPanelShownFunction::Run() {
```

Comment:
this was supposed to be converted to a p3a extension api

Reply:
I'm ok with doing that as a follow-up, but we should not add anymore extension api methods for individual p3a values like this

Reply:
is there an issue created for follow-up?

Reply:
https://github.com/brave/brave-browser/issues/6284